### PR TITLE
feat(config): Add configuration for javascript version. 

### PR DESCRIPTION
**Pull request by Moumi**
Add configuration for a javascript version tag. This tag is only applied to Firefox (now).

Closes *[#1719](https://github.com/karma-runner/karma/issues/1719)*

### CODE REVIEW

File: lib/middleware/karma.js [hunk 1]
```diff
@@ -59,9 +60,21 @@ var getXUACompatibleUrl = function (url) {
   return value
 }
 
+function applyJavascriptVersionTag (req) {
```
Comment (dignifiedquire) [inline lib/middleware/karma.js]: Better name would be `isFirefox` or `hasJsVersionTags`

File: test/e2e/tag.feature [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      """
+    When I start Karma
+    Then it passes with:
+      """
+      ..
+      PhantomJS
+      """
+
+  Scenario: Execute a test in Firefox with version, with JavaScript tag
+    Given a configuration with:
```
Comment (dignifiedquire) [inline test/e2e/tag.feature]: This test will not actually check that the version tag is available, you should add another file that actually checks the html for the version tag

Comment (Moumi) [inline test/e2e/tag.feature]: How would I be able to do this. I am not familiar with e2e tests.

Comment (dignifiedquire) [inline test/e2e/tag.feature]: You can create a new file similar to what is in `basic/test.js`, e.g. `basic/version.js` and include it in the files here. That means that file will be run in the browser. Now in that file you can put some js in that reads the current html and checks for the version tag to be present, and fails if it's not present.

**Comment by Moumi** [comment]
This PR is getting a bit messy. Closing it to create a new one.

**Pull request closed without merging**